Swift/ChartboostExample.xcodeproj/project.pbxproj

```diff
@@ -7,30 +7,30 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		342BBF295473A28B87B0C20A /* Pods_ChartboostExample_Pods.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 62F672CEC8AE474CE80A0CCF /* Pods_ChartboostExample_Pods.framework */; };
```

Comment:
I see a lot of changes in the Swift xcodeproj but none on the Obj-C xcodeproj. Is there a reason for this?

Reply:
yes, there was a missing target in the Swift project